.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  node_version: "12.x"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+        node-version: ${{ env.node_version }}
+      - name: Cache node_modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-node_modules-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node_modules-
+      - name: Install JS dependencies
+        run: yarn install
+      - name: Build
+        run: yarn run distify
+      - name: Commit and push distified version to master
+        uses: stefanzweifel/git-auto-commit-action@v4.1.2
+        with:
+          commit_message: 'Update distified version'
+          file_pattern: 'dist/index.js'
```

Comment:
Reason for no longer passing the `GITHUB_TOKEN` to the action: https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v3.0.0